packages/cw-orch-core/src/environment/queriers/wasm.rs

```diff
@@ -27,6 +28,30 @@ pub trait WasmQuerier: Querier {
         query_keys: Vec<u8>,
     ) -> Result<Vec<u8>, Self::Error>;
 
+    fn item_query<T: Serialize + DeserializeOwned>(
+        &self,
+        address: impl Into<String>,
+        item: Item<T>,
+    ) -> Result<T, CwEnvError> {
+        let current_manager_version = self
+            .raw_query(address, item.as_slice().to_vec())
+            .map_err(Into::into)?;
+
+        from_json(current_manager_version).map_err(Into::into)
+    }
```

Comment:
Should we add to it a method that can query any key (Vec<u8>) to support any types of state objects, including custom?

Reply:
> What do you mean ? This is the raw query function here, it's already available. I have to rename those objects though, wtf are those names haha

I mean on contract object: `item_query(ITEM)`, `map_query(MAP, KEY)` and `raw_state_query(KEY)`(not sure about the name)

Reply:
Ok, done